core/src/main/java/org/stellar/anchor/dto/sep38/InfoResponse.java

```diff
@@ -14,26 +14,28 @@
   public InfoResponse(List<AssetInfo> assetInfoList) {
     assetInfoList.forEach(
         assetInfo -> {
-          Asset newAsset = new Asset();
-          String assetName = assetInfo.getSchema().toString() + ":" + assetInfo.getCode();
-          if (!Objects.toString(assetInfo.getIssuer(), "").isEmpty()) {
-            assetName += ":" + assetInfo.getIssuer();
+          if (assetInfo.getSep38Enabled()) {
```

Comment:
nitpick: I'd prefer an approach without an extra nesting level, and return early instead. Something like:

```java
for (AssetInfo assetInfo : assetInfoList) {
  if (!assetInfo.getSep38Enabled())
    continue;

  // TODO: continue doing what we were doing.
```

Reply:
I also like this better!